homeassistant/components/homekit_controller/config_flow.py

```diff
@@ -141,6 +148,17 @@ async def async_step_unignore(self, user_input):
 
         return self.async_abort(reason="no_devices")
 
+    async def _hkid_is_homekit_bridge(self, hkid):
+        """Determine if the device is a homekit bridge."""
+        dev_reg = await async_get_device_registry(self.hass)
+        device = dev_reg.async_get_device(
+            identifiers=set(), connections={(CONNECTION_NETWORK_MAC, hkid)}
```

Comment:
Does this work? Hkid is not a MAC address, so I thought i used a different label for this identifier when creating the device registry entry. Might be misremembering?

Reply:
Yes it does. HAP-python presents the `id` as a mac address and `homekit` uses it that way. It is likely an artifact of assumptions made before Apple published the spec that we have held on to.

Reply:
Ah of course! I saw hkid and latched on to hkc's config entries, but of course hk doesn't have the same identifiers. D'oh. That's what I get for reviewing on my phone when I've already been up too long.